src/structuredLogger.js

```diff
@@ -10,69 +10,56 @@ function write(logData) {
   process.stdout.write(`${logData}\n`);
 }
 
-function createLogObject(metadata) {
-  const logObject = {
-    Stack: stack
-  };
-
-  if (!metadata) {
-    return logObject;
-  }
+function createlogData(message, args, logLevel = 'INFO') {
+  const safeArgs = args instanceof Object ? args : {};
+  const metadata = Object.assign(safeArgs || {}, baseMetadata);
```

Comment:
seems like `safeArgs` will always be valid, so how about:
```
const metadata = {
   ...baseMetadata,
   ...safeArgs
};

Reply:
Even if args is a simple string?

Reply:
well, a string is not `instanceof Object` (unless you do `new String('')` maybe)
 so `safeArgs` is either `args` when it's an object, or it's `{ }`, so I'm pretty sure the right hand side of `||` will never run

Reply:
ah, good point. feel free to adjust it :)